src/process/chem/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+set(_lib CATChem_process_chem)
+
+set(_srcs CCPr_Chem_Mod.F90)
+
+add_library(${_lib} ${_srcs})
```

Comment:
The only cmake related comment I can think of is that it's recommended to use (`target_sources`)[https://cmake.org/cmake/help/latest/command/target_sources.html] in newer versions of cmake. I think if you use `target_sources` and generate a project for something like Xcode or Visual Studio, this helps Cmake create better integration with that IDE. This certainly isn't a requirement though

Reply:
Thanks @K20shores , I can update the whole project to use that in a separate PR.